### PR TITLE
 Added Neumorphism_Smarthome_Darkmode

### PR DESCRIPTION
**Pull request by arynn-gupta**

### CODE REVIEW

File: entry/src/main/java/com/cyb0rg/neumorphism_smart_home_dark/MainAbility.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package com.cyb0rg.neumorphism_smart_home_dark;
```
Comment (archanaatr) [inline entry/src/main/java/com/cyb0rg/neumorphism_smart_home_dark/MainAbility.java]: Include copyright in all new files

File: entry/src/main/js/default/i18n/en-US.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (archanaatr) [inline entry/src/main/js/default/i18n/en-US.json]: Remove unused strings

File: entry/src/main/js/default/pages/index/index.hml [hunk 3]
```diff
@@ -0,0 +1,361 @@
+<div class="body body_bg">
+    <div class="container container_bg">
+        <div class="main">
+            <div class="user">
```
Comment (archanaatr) [inline entry/src/main/js/default/pages/index/index.hml:21]: Request to use these required components and update the application.
https://github.com/applibgroup/Theme
https://github.com/applibgroup/Alert
https://github.com/applibgroup/Buttons
https://github.com/applibgroup/Card
https://github.com/applibgroup/Checkbox
https://github.com/applibgroup/Dropdown
https://github.com/applibgroup/form
https://github.com/applibgroup/Navbar
https://github.com/applibgroup/pagination
https://github.com/applibgroup/progress

**Review by archanaatr — COMMENTED** [review]
Pls check the comments

**Review by superlib-framework — APPROVED** [review]
Ok

**Comment by archanaatr** [comment]
Ok

**Pull request merged**